web/.ht.router.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
I think we should add this to the .gitignore. We don't need this file in our project.

Reply:
According to https://github.com/drupal-composer/drupal-scaffold this is a "scaffold file" and

> It is assumed that the scaffold files will be committed to the repository, to ensure that the correct files are used on the CI server (see Limitation, above). After running composer install for the first time commit the scaffold files to your repository.

Not needing that file it's not a good reason. We have also other (scaffold) files, like `web.config`, that we are not using but we are still keep them to ensure the integrity. I would not start to check file by file and see that we are using or not. I prefer that we have a complete Drupal codebase.

Reply:
Indeed you are right on that! Didn't think it that way! I thought they were there by mistake :P (not the web.config since it is a server file but the rest)